### PR TITLE
Add pytorch-randaugment package

### PR DESCRIPTION
**Pull request by sthalles**
FYI @hyviquel

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.  Adding @adamjstewart since he is the python package maintainer.

File: var/spack/repos/builtin/packages/pytorch-randaugment/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PytorchRandaugment(PythonPackage):
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pytorch-randaugment/package.py:9]: Package name should be `py-randaugment`

File: var/spack/repos/builtin/packages/pytorch-randaugment/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    """Unofficial PyTorch Reimplementation of AutoAugment and RandAugment."""
+
+    homepage = "https://github.com/jizongFox/pytorch-randaugment"
+    url = "https://files.pythonhosted.org/packages/ac/a3/0d76507a2048b38fe9cc77f85b8ac008543b0c2c237feb5a1a1a882d46aa/randaugment-1.0.2.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pytorch-randaugment/package.py:13]: https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#id2

File: var/spack/repos/builtin/packages/pytorch-randaugment/package.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    maintainers("sthalles")
+
+    version("1.0.2", sha256="7f2dda16f9daef2d5d76d376b63a31154a7e4d7d5a562e552ce1fbf4a8409566")
+
+    depends_on("py-setuptools", type="build")
+    depends_on("python", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pytorch-randaugment/package.py:20]: This is already added by the base class and can be removed

File: var/spack/repos/builtin/packages/pytorch-randaugment/package.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    depends_on("py-torch", type=("build", "run"))
+    depends_on("py-torchvision", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/pytorch-randaugment/package.py:22]: Package does not depend on torch/torchvision

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by alecbcs** [comment]
Going to close this PR due to a lack of response after review feedback. @sthalles happy to reopen if you still want to get this merged and have time to work on it.

**Pull request closed without merging**